lib/bitcoin/models/LockTransactionModel.ts

```diff
@@ -3,9 +3,11 @@ import LockTransactionType from '../enums/LockTransactionType';
 /**
  * Encapsulates the data about a bitcoin 'lock' transaction saved in the database.
  */
-export default interface SavedLockTransactionModel {
+export default interface LockTransactionModel {
   transactionId: string;
-  redeemScript: string;
+  rawTransaction: string;
+  redeemScriptAsHex: string;
+  desiredLockAmountInSatoshis: number;

```

Comment:
Can it be just:
```suggestion
  lockAmountInSatoshis: number;

```

Reply:
Well remember that the actual lock amount is different from the 'desired' amount. And we want to save the 'desired' amount so that we'll know if the admin changes the desired amount.

Reply:
Got it!